app/till.rb

```diff
@@ -0,0 +1,16 @@
+require 'json'
+
+class Till
+
+file = File.read('./hipstercoffee.json')
+SHOP_DETAILS = JSON.parse(file)
+
+  def initialize()
```

Comment:
Style/DefWithParentheses: Omit the parentheses in defs when the method doesn't accept any arguments. (https://github.com/bbatsov/ruby-style-guide#method-parens)